azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -699,12 +702,16 @@ private void uploadLogFile(final int execId) {
    */
   @Override
   public void handleEvent(final Event event) {
-    final FlowRunner flowRunner = (FlowRunner) event.getRunner();
-    final ExecutableFlow flow = flowRunner.getExecutableFlow();
-    // Set Flow level SLA options for containerized executions
-    this.triggerManager
-        .addTrigger(flow.getExecutionId(), SlaOption.getFlowLevelSLAOptions(flow
-            .getExecutionOptions().getSlaOptions()));
+    final List<EventType> flowEventTypeList = Arrays.asList(EventType.FLOW_STARTED,
```

Comment:
Why don't you take this in enum? Introduce method like isFlowEvent(eventType)

Reply:
good idea